src/Platform/Microsoft.Testing.Extensions.VSTestBridge/ObjectModel/ObjectModelConverters.cs

```diff
@@ -167,11 +167,13 @@ public static TestNode ToTestNode(this TestResult testResult, bool isTrxEnabled,
             if (testResultMessage.Category == TestResultMessage.StandardErrorCategory)
             {
                 testNode.Properties.Add(new SerializableKeyValuePairStringProperty("vstest.TestCase.StandardError", testResultMessage.Text ?? string.Empty));
+                testNode.Properties.Add(new StandardErrorProperty(testResultMessage.Text ?? string.Empty));
```

Comment:
These are potentially much larger than the rest of the message, is it good idea to have 2 copies of the text in the object? Is this the only way to do this? can `"vstest.TestCase.StandardError"` be dropped in favor of the new Platform provided property?

Reply:
We cannot we need 2 way because the `SerializableKeyValuePairStringProperty` is an internal prop to accomodate VS. VS TE is working in 2 different mode where the values are taken using different key, value. 
Adding `StandardErrorProperty` like xunit did is to have the new model and avoid the need to go with an untyped "key name" that's also used in the protocol atm as vstest capability.

Reply:
My idea was to drop the vstest capability, but looks like at time wasn't good for TE. cc: @drognanar

Reply:
So the solution here is fix the VS client, and then remove the vstest.* property, yeah?

Reply:
Do you know if we end up serializing both when calling TE or some other client? If this is not the case then the overhead of having two "copies" of the same string on the node is minimal, but I doubt that is the case, and we actually send it twice. 

If you don't know off hand just say it , I will make an issue for investigating it.

Reply:
created: #3933